content/spin/index.md

```diff
@@ -12,7 +12,7 @@ Spin uses Wasm because it is **sandboxed, portable, and fast**.  Millisecond col
 
 Many languages have Wasm implementations, so **developers don't have to learn new languages or libraries**.
 
-Spin is **open source** and **built on standards**, meaning you can take your Spin applications anywhere.  There are Spin implementations for local development, for self-hosted servers, for Kubernetes, and for cloud-hosted services.
+Spin is **open source** and **built on standards**, meaning you can take your Spin applications anywhere.  There are [Spin implementations for local development](see-what-people-have-built-with-spin), for self-hosted servers, for Kubernetes, and for cloud-hosted services.
```

Comment:
suggestion to decouple "built with" from "local development" and make it more of a zingy standout

```suggestion
Spin is **open source** and **built on standards**, meaning you can take your Spin applications anywhere.  There are Spin implementations for local development, for self-hosted servers, for Kubernetes, and for cloud-hosted services.

**Want to see the kinds of things people are building with Spin?**  Check out what's [Built With Spin](/spin/see-what-people-have-built-with-spin)!

Or dive into the documentation and get started:
```

"zingy standout" what have I become Tim